src/routes/api_docs.ts

```diff
@@ -0,0 +1,26 @@
+import type { Router } from "express";
+import swaggerUi from "swagger-ui-express";
+import { readFileSync } from "fs";
+import { fileURLToPath } from "url";
+import { dirname, join } from "path";
+import yaml from "js-yaml";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const swaggerDocument = yaml.load(
+    readFileSync(join(__dirname, "../etapi/etapi.openapi.yaml"), "utf8")
+) as object;
+
+function register(router: Router) {
+    router.use(
+        "/api-docs",
+        swaggerUi.serve,
+        swaggerUi.setup(swaggerDocument, {
+            explorer: true,
+            customSiteTitle: "Trilium ETAPI Documentation"
```

Comment:
I wonder, if that should say `TriliumNext ETAPI Documentation`?

Not sure how strict we need to be with the naming difference, @eliandoran ?